packages/sdk/package.json

```diff
@@ -27,7 +30,8 @@
     "test": "jest --collect-coverage",
     "build:docs": "jsdoc -c jsdoc.json",
     "pre-commit": "yarn lint",
-    "pre-push": "yarn test"
+    "pre-push": "yarn test",
+    "build:typechain": "typechain --target ethers-v5 --out-dir ./src/contracts './artifacts/*.json' './node_modules/@artblocks/contracts/artifacts/**/*.json'"
```

Comment:
This is roughly the same approach taken by the Art Blocks web site for generating types - @yoshiwarab does this look good to you?

Reply:
Done!